packages/validator-rules/index.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const loadRules = require('./lib/loadRules');
+const AmpValidatorRules = require('./lib/AmpValidatorRules');
+
+let cached = null;
+
+async function load(opt) {
+  opt = opt || {};
+  if (!opt.noCache && cached) {
+    return cached;
+  }
+
+  let rules = opt.rules;
+  delete opt.rules;
```

Comment:
Why is this needed? It's also potentially dangerous to modify the object that passed from the outside.

Reply:
I don't even remember why I had it there. Thanks for flagging, removed.